Classes/BEMSimpleLineGraphView.m

```diff
@@ -1089,13 +1093,14 @@ - (void)displayPermanentLabelForPoint:(BEMCircle *)circleDot {
     NSString *prefix = @"";
     NSString *suffix = @"";
     
-    if ([self.delegate respondsToSelector:@selector(popUpSuffixForlineGraph:)])
-        suffix = [self.delegate popUpSuffixForlineGraph:self];
+    int index = (int)(circleDot.tag - DotFirstTag100);
```

Comment:
Why casting to an int here?
`    NSInteger index = (circleDot.tag - DotFirstTag100);` would cause problems?